ogre_mesh_viewer.py

```diff
@@ -290,17 +292,27 @@ def preRenderTargetUpdate(self, evt):
 
         lod_count = mesh.getNumLodLevels()
         if lod_count > 1 and ImGui.CollapsingHeader("LOD levels"):
-            entity.setMeshLodBias(1)  # reset LOD override
+            entity.setMeshLodBias(1, self.lod_current_idx, self.lod_current_idx)  # reset LOD override
             strategy = mesh.getLodStrategy().getName()
             curr_idx = entity.getCurrentLodIndex()
             ImGui.Text("Strategy: {}".format(strategy))
+
+            lod_levels = []
             for i in range(lod_count):
-                txt = "Base Mesh" if i == 0 else "Level {}: {:.2f}".format(i, mesh.getLodLevel(i).userValue)
-                ImGui.Bullet()
-                ImGui.Selectable(txt, i == curr_idx)
-                if ImGui.IsItemHovered():
-                    # force this LOD level
-                    entity.setMeshLodBias(1, i, i)
+                lod_levels.append("Base Mesh" if i == 0 else "Level {}: {:.2f}".format(i, mesh.getLodLevel(i).userValue))
+
+            if ImGui.BeginListBox("##lod_levels", ImGui.ImVec2(-1, min(3, lod_count) * ImGui.GetTextLineHeightWithSpacing())):
+                for n in range(lod_count):
+                    is_selected = (self.lod_current_idx == n)
+                    if ImGui.Selectable(lod_levels[n], is_selected):
+                        self.lod_current_idx = n
+                        entity.setMeshLodBias(1, n, n)
+                    # Set the initial focus when opening the combo (scrolling + keyboard navigation focus)
+                    if is_selected:
+                        ImGui.SetItemDefaultFocus()
+                    if ImGui.IsItemHovered():
```

Comment:
I dont think that we can drop this one now

Reply:
Without that you can't do a quick inspection of the LOD levels like before